Chap_Terms.tex

```diff
@@ -130,8 +136,8 @@ \section{Naming Conventions}
 The \ac{PMIx} standard has adopted the following conventions:
 
 \begin{itemize}
-\item \ac{PMIx} constants and attributes are prefixed with \textbf{\code{PMIX}}.
-\item Structures and type definitions are prefixed with \code{pmix}.
+\item \ac{PMIx} constants and attributes are prefixed with \textbf{\code{PMIX_}}.
+\item Structures and type definitions are prefixed with \code{pmix_}.
```

Comment:
Per ASC Meeting Oct. 1, 2020:
 * Concern that existing libraries in deployment will have backward compatibility issues. 
   * Because if a user says "PMIXa" - existing libraries only check for "PMIX" thus classifying this attribute as reserved. If we specify this as "PMIX_" then older libraries (per this change) then those deployed libraries would not be able to 
 * We should add a rationale so we don't forget about this at a later date. Something like "all constants/attributes begin with `PMIX_`. However, constants/attributes starting with `PMIX` may be classified as reserved by older libraries. As such we cannot add the `_` in the specification to maintain correctness with implementations of prior standards."